Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md

```diff
@@ -136,3 +136,36 @@ Example:
 }
 <h3>@item.Value("heading")</h3>
 ```
+
+## Creating Nested Content programmatically
+
+For the sake of this example, let us assume we have a Nested Content property with alias `attendeeList`, where the element Document Type has an alias of `attendee`.
+It has the Properties: `user_name`, `user_email`, `join_time`, `leave_time`, `duration`, `phone`.
+
+To save data in Nested Content, we need to create a new C# List containing a `Dictionary` of type `<string, object>`. `Dictionary<string, string>` would also work.
+The first dictionary item property/parameter we should specify for each Nested Content element is `ncContentTypeAlias`, which is the alias of the Document Type that is the embed element in Nested Content.
+
+Afterwards, the entire list needs to be serialized to Json via JsonConvert.
+
+After injecting [ContentService](../../../../../Reference/Management/Services/ContentService/) and creating a new Content Node, we can do the following:
+
+```csharp
+ var contentService = Services.ContentService; //if the class containing our code inherits SurfaceController, UmbracoApiController, or UmbracoAuthorizedApiController, we can get ContentService from Services namespace
+ IContent request = ContentService.Create("new node", guid, "mydoctype", -1); //here we create a new node, and fill out attendeeList afterwards
+ var attendees = new List<Dictionary<string, string>>(); //our list which will contain nested content
+ foreach (var person in participants) //participants is our list of attendees - multiple items, good use case for nested content
+            attendees.Add(new Dictionary<string, string>() {
+            {"ncContentTypeAlias","attendee"}, //this is the only "default" value we need to fill for nested item
```

Comment:
Sounds like this is something that is required when working with Nested Content - is that correct?
If that's indeed the case, perhaps this `ncContentTypeAlias` should be explained a bit more? 🤔

Reply:
ncContentTypeAlias is just the alias of the document type - can I explain it this way

Reply:
Ah. But it's the name of the alias when it is a Nested Content Element type - right?